app/services/plugins/authentication.service.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+/**
+ * Used by `AuthenticationPlugin` to authenticate and authorise users
+ * @module AuthenticationService
+ */
+
+const FetchUserRolesAndGroupsService = require('../idm/fetch-user-roles-and-groups.service.js')
+
+/**
+ * This service is intended to be used by our `AuthenticationPlugin` to authenticate and authorise users.
+ *
+ * We take a user id and look it up in the `idm` table using `FetchUserRolesAndGroupsService`. This gives us a user
```

Comment:
I think this is what you mean. 🤔 

```suggestion
 * We take a user id and look it up in the `idm` schema using `FetchUserRolesAndGroupsService`. This gives us a user
```